priv/beam.json

```diff
@@ -493,7 +489,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "beam_memory_total_kilobytes{job=\"$job\", instance=\"$instance\"}",
+          "expr": "beam_memory_allocated_kilobytes{job=\"$job\", instance=\"$instance\"}",
```

Comment:
I keep seeing beam in a couple of places, and I think that now would be a good time to talk about that.

BEAM is an Erlang implementation, the one that we use today. If this was Java, BEAM would be like Graal, a JVM implementation. If you agree that `graal_` makes less sense than `java_` or just `jvm_`, then we should do something about this.

I know that this is going to be a pain, but I would replace all occurences of `beam_` with `erlang_`. Good thing this is still pre `1.0`.